tablestore/translatetoKV_test.go

```diff
@@ -3,7 +3,7 @@ package tablestore
 import (
     "testing"
    // "github.com/stretchr/testify/assert"
-    //"fmt"
+   // "fmt"
 )
 
 func TestTranslateToKV(t *testing.T) {
```

Comment:
Make sure tests actually test different scenarios for correctness. Why are all assertions commented out?